### PR TITLE
added next and previous links to articles

### PR DESCRIPTION
**Pull request by GarthDB**

### CODE REVIEW

**Comment by GarthDB** [comment]
image

**Comment by GarthDB** [comment]
image

image

File: _sass/_footer.scss [hunk 1]
```diff
@@ -26,6 +26,15 @@
   border-top: 1px solid #EEE;
   text-align: center;
   box-sizing: border-box;
-  padding: 10px;
+  padding: 10px 10px 30px 10px;
   margin-top: 30px;
+  position: relative;
+  a.next {
```
Comment (una) [inline _sass/_footer.scss]: selectors with specified classes should not start with their base elements. This should just be `.next` :)

File: _sass/_footer.scss [hunk 2]
```diff
@@ -26,6 +26,15 @@
...
+    position: absolute;
+    left: 20px;
+  }
+  a.prev {
```
Comment (una) [inline _sass/_footer.scss]: also `.prev`

**Pull request merged**